src/modules/comment/popup.js

```diff
@@ -0,0 +1,78 @@
+import postComment, { getComments } from './postComment.js';
+
+export const checkImage = (image) => { // eslint-disable-line
+  if (image) {
+    return image.original;
+  }
+};
+
+const closeModal = () => {
+  const modalContainer = document.getElementById('modal-container');
+  const modal = document.querySelector('.modal');
+  const backdrop = document.querySelector('.modal-backdrop');
+  modalContainer.removeChild(modal);
+  modalContainer.removeChild(backdrop);
+};
```

Comment:
#### Great job on your comment popup team. But I suggest you do the following. 
- [ ] When a user clicks on the "Comment" button, beside the data being recorded to the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270), the screen should be updated automatically with the new comment.
- [ ] Also, Please add a visible `close` button or icon. Currently, I had to double-click on the free space to get rid of the pop-up. this is against the wireframe and not good when it comes to UX.
- [ ] There should be a comment counter on the pop-up window, and the value should be incremented accordingly. when we add a new comment the comment count should be incremented and the screen is populated with the new count. and remember, the counter should be implemented as a separate module.
- [ ] Once we have the comment counter, we need to cover the counter function with unit tests using Jest. Make sure that you have tests for all edge cases

https://user-images.githubusercontent.com/69077061/221371093-de276cbc-974e-4119-90de-513642e7f44c.mp4

Reply:
Fixed.